test/functional/autocmd/termclose_spec.lua

```diff
@@ -20,8 +20,8 @@ describe('TermClose event', function()
   it('triggers when fast-exiting terminal job stops', function()
     command('autocmd TermClose * let g:test_termclose = 23')
     command('terminal')
-    command('call jobstop(b:terminal_job_id)')
     retry(nil, nil, function() eq(23, eval('g:test_termclose')) end)
+    assert(-1 ~= eval('jobwait([&channel], 0)[0]'))
```

Comment:
should go above line 23, to verify the semantics of the `g:test_termclose` assert.

Reply:
ok, done.